api/turing/models/alert.go

```diff
@@ -146,20 +146,28 @@ func getAlertExpr(metric Metric, env string, rev string, threshold float64) stri
   service_name=~"%s-[0-9]*"
 }[1m])) by (le)) %s %f
 `, env, rev, getAlertOperator(metric), threshold)
-
 	case MetricErrorRate:
-		return fmt.Sprintf(`sum(rate(revision_request_count{
```

Comment:
What about the `revision_request_latencies_bucket` metric with UPI? Is it expected to still work, or does it have to be changed?